### DISCUSSION TITLE
Welcome to CoinCanvas Discussions!

### DISCUSSION THREAD

**Discussion by TomasAndersonFang** [DFY-NCSU/CoinCanvas#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomasAndersonFang** [comment]
@IMYXR @TheFatBlue Welcome to discussion in this channel.

**Comment by TomasAndersonFang** [comment]
There are two basic rules:
- Please create issue reports from our [project](https://github.com/orgs/DFY-NCSU/projects/2/views/1)
- For any feature you want to add, please create a feature branch to work.

**Comment by TheFatBlue** [comment]
@IMYXR We have finished the initial implementation of the backend and test it. Currently the frontend files are just a demo and cannot be run. Can you add a concrete implementation for it? For the usage of backend, you can refer [here](https://github.com/DFY-NCSU/CoinCanvas/issues/3#issuecomment-2477509247)

**Comment by TheFatBlue** [comment]
@TomasAndersonFang @IMYXR Hey guys! 👋

Our recent `flake8` check in the workflow failed due to a few common formatting issues (also see #17):

- **E302**: Expected 2 blank lines before a function definition, but found 1.
- **W292**: No newline at the end of the file.
- **W293**: Blank line contains whitespace.

These are small things, but they make a big difference in keeping our code clean and professional! 💻✨ Let’s fix the existing issues and make sure we’re all sticking to good habits moving forward.

A couple of quick tips:

- Use an editor or IDE that supports auto-formatting (like VSCode or PyCharm).
- Run `flake8` locally before pushing changes.

Let me know if you have other tips or tools to share! Let's work together to keep things neat and tidy. 🙌

**Reply by TomasAndersonFang** [reply]
Thanks for your reminder and suggestions, I'll follow this standard.

**Reply by IMYXR** [reply]
Thanks for your reminder!
